src/app/service/theme.service.ts

```diff
@@ -31,8 +31,11 @@ export class ThemeService {
     private init() {
         const savedTheme = this.settingStorage.getSetting(Setting.THEME) as Theme;
         const prefersColorSchemeDark = window.matchMedia("(prefers-color-scheme: dark)");
-        let useDarkMode = (prefersColorSchemeDark.matches || Theme.DARK == savedTheme) || savedTheme == null;
-        let themeToApply: Theme = (useDarkMode) ? Theme.DARK : savedTheme;
+
+        let themeToApply: Theme;
+        if (savedTheme == null) {
```

Comment:
Could use `if (!savedTheme) { ... }`, as `null` is 'falsy' in JavaScript

Reply:
Thanks. I'm going to apply this :)